spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/kafka/KafkaPropertiesTests.java

```diff
@@ -128,42 +114,6 @@ void sslPropertiesWhenTrustStoreLocationAndCertificatesSetShouldThrowException()
 			.isThrownBy(properties::buildConsumerProperties);
 	}
 
-	@Test
-	void sslPropertiesWhenKeyStoreLocationAndBundleSetShouldThrowException() {
```

Comment:
This test should be kept but should call `.isThrownBy(properties::buildConsumerProperties);`.